### PR TITLE
added method YandexCleanweb.spam!

### PR DESCRIPTION
**Pull request by r00takaspin**
Added method YandexCleanweb.spam!, that allows tell api what messages are spam

### CODE REVIEW

File: lib/yandex_cleanweb.rb [hunk 1]
```diff
@@ -97,6 +123,15 @@ def api_check_spam(options)
       response.body
     end
 
+    def api_mark_as_spam(request_id)
+      cleanweb_options = { key: prepare_api_key, id:request_id,spamtype: 'spam' }
+      complain_url = "#{API_URL}/complain"
+
+      uri = URI.parse(complain_url)
+      response = Net::HTTP.post_form(uri,cleanweb_options)
```
Comment (kirs) [inline lib/yandex_cleanweb.rb]: Could you add a whitespace between arguments?

File: lib/yandex_cleanweb.rb [hunk 2]
```diff
@@ -97,6 +123,15 @@ def api_check_spam(options)
       response.body
     end
 
+    def api_mark_as_spam(request_id)
+      cleanweb_options = { key: prepare_api_key, id:request_id,spamtype: 'spam' }
```
Comment (kirs) [inline lib/yandex_cleanweb.rb]: The same about whitespaces here

File: lib/yandex_cleanweb/version.rb [hunk 3]
```diff
@@ -1,3 +1,3 @@
 module YandexCleanweb
-  VERSION = "0.0.6"
+  VERSION = "0.0.7"
```
Comment (kirs) [inline lib/yandex_cleanweb/version.rb]: I will bump the version in a separate commit. Can you remove this change, please?

Comment (r00takaspin) [inline lib/yandex_cleanweb/version.rb]: done

File: lib/yandex_cleanweb.rb (hunk 2 above)
Comment (r00takaspin) [inline lib/yandex_cleanweb.rb]: done

File: lib/yandex_cleanweb.rb (hunk 1 above)
Comment (r00takaspin) [inline lib/yandex_cleanweb.rb]: done

**Pull request merged**